### DISCUSSION TITLE
Getting error when installing Homebrew on Mac M1 using Roseeta

### DISCUSSION THREAD

**Discussion by iakashkanaujiya** [Homebrew/discussions#1068] [Getting started] [answered]
I am running the command "/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
to install the Homebrew using Roseeta, but getting this error 

HEAD is now at 98b83064e Merge pull request #10871 from Bo98/cask_slash_audit
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

**Comment by SMillerDev** [comment, marked as answer]
Before that there is likely the error discussed here: https://github.com/Homebrew/discussions/discussions/666

**Reply by iakashkanaujiya** [reply]
Thanks
